.github/workflows/continuous-integration.yml

```diff
@@ -2,11 +2,7 @@ name: "Continuous Integration"
 
 on:
   pull_request:
-    branches:
-      - "*.x"
```

Comment:
Why is this removed, seems irrelevant?

Reply:
Not really. I took the [branch](https://github.com/doctrine/orm/pull/8958) as a basis and it was already deleted there. I'll return when I'm done

Reply:
As long as this is WIP, I understand that this change can be helpful. Let's keep this comment here as a reminder that we need to revert before merging.

Reply:
> As long as this is WIP, I understand that this change can be helpful. Let's keep this comment here as a reminder that we need to revert it before merging.

returned as it was